packages/example/src/data/nav-items.yaml

```diff
@@ -22,6 +22,8 @@
       path: /guides/styling
     - title: Navigation
       path: /guides/navigation/sidebar
+    - title: Navigation
```

Comment:
rename to scripts or NPM scripts or something like that..
![image](https://user-images.githubusercontent.com/15822070/73767004-77b23900-473c-11ea-87d2-dea102796634.png)

Reply:
I hate how they don't capitalize 😢